package.json

```diff
@@ -27,7 +27,6 @@
     "coreThemeBlocksDir": "./wp-content/themes/core/blocks"
   },
   "devDependencies": {
-    "@csstools/postcss-global-data": "^1.0.3",
```

Comment:
Ah, 💩 . I must have made this unclear in my Slcak convo. I'm hoping to still use this plugin, but not using a `/globals/` directory for items to include. Instead, we can reference the variable & mixin partials across any number of directories using this plugin.

So, for example, we could still set up the color partials as you have already, but we can add `${ pkg.config.coreThemeDir }/assets/pcss/**/_variables.pcss` or maybe we should be more specific to only the partials we want to use.

Reply:
Additionally, Imho, I think we can skip the `global` directory all together. ANy styles specific to a component, block, pattern, or template should live in the same directory as the markup & other respective files for that item. (Similar to Sq1's components.)

Reply:
@dpellenwood Gotcha, that makes sense re: use of the globals postcss plugin. I'll update.

What are your thoughts on renaming /assets/pcss/**global**/ to /assets/pcss/**theme**/? That directory would hold things like color, custom selectors, media queries, type mixins, etc.. things that need to be setup for the general theme. It wouldn't include styles specific to blocks/components.

On [LendNation](https://github.com/wpcomvip/qchi-lendnation/tree/develop/themes/core/assets/pcss), we needed to include `/patterns/*.pcss` and `/templates/*.pcss` at the same level as `/assets/pcss/global` because we didn't have a way to load style partials for patterns or templates individually like blocks. I assumed we'd need to do the same here, so the structure would be something like:
```
pcss/
- theme/
--- color
--- custom-selectors
--- media-queries
--- typography

- patterns/
--- hero

- templates/
--- 404
--- single 
```

Reply:
I think we can avoid global and/or theme. as the css in the assets directory will be only that. Totally get how it was set up on LendNation. We're planning to make it possible to enqueue the css for templates/patterns/components from within those respective directories here. There's a backend ticket for that in the works already. :)

Reply:
You configured the plugin correctly. any css defined in a `_variables.pcss` file will now be available to use in any other processed pcss file, even if the variables file(s) were not specifically imported, yes.